### PR TITLE
Proposal: Add pre check logic for crowdin update work

### PR DESCRIPTION
**Pull request by a1012112796**
Add check step to make sure that the translation result's not has format error.

### CODE REVIEW

**Comment by zeripath** [comment]
So is this going to write to the #drone channel on discord?

**Comment by a1012112796** [comment]
> So is this going to write to the #drone channel on discord?

yes, It's just be copyed from other place in this file :). In my opion, it's better to comment in ``#translations`` channel. But this plugin looks not support...

**Comment by jolheiser** [comment]
It sends to the channel specified by `webhook_id` and `webhook_token`

File: build/test-locales.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+import (
+	"errors"
+	"fmt"
+
+	"log"
+
+	"code.gitea.io/gitea/modules/options"
+	"code.gitea.io/gitea/modules/setting"
+	"github.com/unknwon/i18n"
```
Comment (6543) [inline build/test-locales.go]:
```suggestion
	"code.gitea.io/gitea/modules/setting"

	"github.com/unknwon/i18n"
```

**Review by 6543 — APPROVED** [review]

**Comment by 6543** [comment]
> It sends to the channel specified by `webhook_id` and `webhook_token`

I'm for this CI task :D

... if it depends on webhoock_id + token | create just a specific one for the #translation channel?

File: build/test-locales.go [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+	localFiles := make(map[string][]byte)
+	for _, name := range localeNames {
+		localFiles[name], err = options.Locale(name)
+		if err != nil {
+			log.Fatal(fmt.Sprintf("Failed to load %s locale file. %v", name, err))
+		}
+	}
+
+	setting.Langs = []string{
```
Comment (lunny) [inline build/test-locales.go:38]: But languages may be changed in future.

**Comment by a1012112796** [comment]
Close this pull request because it's not so necessary.

**Pull request closed without merging**